appinfo/info.xml

```diff
@@ -2,10 +2,18 @@
 <info>
     <id>contacts</id>
     <name>Contacts</name>
-    <summary>A contacts app for ownCloud</summary>
+    <summary>Manage your CardDAV contacts from a feature-rich web-interface</summary>
     <description>
-The ownCloud contacts app is a user interface for ownCloud's CardDAV server.
-Easily sync contacts from various devices with your ownCloud and edit them online.</description>
+Contacts is a user interface for ownCloud's CardDAV server. ownCloud Server comes with a CardDAV component that allows users to synchronize and backup their address books with ownCloud and across multiple devices.
+Thanks to this feature users are provided with an alternative to proprietary contact synchronization services giving them back control over their data once more.
+
+The Contacts application is an extension to ownCloud's CardDAV interface. It equips ownCloud with a feature-rich frontend for the CardDAV server component, supports multiple address books and empowers users to easily manage their contacts from a web interface for ownCloud.
```

Comment:
no, it's a separation between the server component and the capabilities of the app